src/query/graphite/native/engine.go

```diff
@@ -32,6 +34,12 @@ type Engine struct {
 
 // NewEngine creates a new query engine.
 func NewEngine(store storage.Storage) *Engine {
+	// Enable pooling.
+	bucket := []pool.Bucket{
+		pool.Bucket{Capacity: 16, Count: 4098},
```

Comment:
Maybe we should enable a few other buckets and/or make this flow from config? (can punt on the second for another diff if that makes sense)

Just for reference, a single time series queried for 2 days retention at 10 second resolution would require 17,280 capacity `(2*24*60*6)`.

Reply:
Yeah fair call, figured something was better than nothing, will bump it up for now and cut the count down.